src/main/scala/no/ndla/searchapi/controller/NdlaController.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Part of NDLA search_api.
+ * Copyright (C) 2017 NDLA
+ *
+ * See LICENSE
+ *
+ */
+
+
+package no.ndla.searchapi.controller
+
+import javax.servlet.http.HttpServletRequest
+
+import no.ndla.network.{ApplicationUrl, AuthUser, CorrelationID}
+import no.ndla.searchapi.SearchApiProperties.{CorrelationIdHeader, CorrelationIdKey}
+import no.ndla.searchapi.model.api.{Error, ValidationException, ValidationMessage}
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.logging.log4j.ThreadContext
+import org.json4s.native.Serialization.read
+import org.json4s.{DefaultFormats, Formats}
+import org.scalatra._
+import org.scalatra.json.NativeJsonSupport
+
+import scala.util.{Failure, Success, Try}
+
+abstract class NdlaController extends ScalatraServlet with NativeJsonSupport with LazyLogging {
```

Comment:
Mye av dette føles som fellesting, som vi kunne skilt ut i et bibliotek og gjenbrukt for alle API'er... Eller?